src/plugins/controls/public/control_group/external_api/control_group_renderer.tsx

```diff
@@ -41,7 +42,8 @@ export interface ControlGroupRendererProps {
   filters?: Filter[];
   getCreationOptions?: (
     initialInput: Partial<ControlGroupInput>,
-    builder: ControlGroupInputBuilder
+    builder: ControlGroupInputBuilder,
+    fieldFilterPredicate?: FieldFilterPredicate
```

Comment:
```suggestion
```

It seems like you should remove this as when you call this function, you don't provide any value for this argument. Plus the consumer should be providing this to the renderer (as they already do), not the other way around.

Reply:
@ThomThomson , sorry. This was a mistake. Removed it now.